src/test/regress/flaky_tests.md

```diff
@@ -0,0 +1,227 @@
+ (13 rows)
+```
+
+**Example fix**:
+
+```diff
+-ORDER BY logicalrelid, shardminvalue::BIGINT;
++ORDER BY logicalrelid, shardminvalue::BIGINT, nodeport;
+```
+
+
+### Expected logs but different order
+
+
+**Issue**: The logs in the regress output are displayed in a different order
+than what the output file shows
+
+**Fix**: It's simple: don't log these things during the test. There's two common
+ways of achieving this:
+1. If you don't care about the logs for this query at all, then you can change
+   the log `VERBOSITY`
+2. If this is are logs created by `citus.log_remote_commands` and you care about
+   some of the remote commands being as expected, but not the ones in question.
+   Then you can use `citus.grep_remote_commands` to only display the commands
+   that you care about.
+
+
+
+**Example 1**:
+```diff
+select alter_table_set_access_method('ref','heap');
+ NOTICE:  creating a new table for alter_table_set_access_method.ref
+ NOTICE:  moving the data of alter_table_set_access_method.ref
+ NOTICE:  dropping the old alter_table_set_access_method.ref
+ NOTICE:  drop cascades to 2 other objects
+-DETAIL:  drop cascades to materialized view m_ref
+-drop cascades to view v_ref
++DETAIL:  drop cascades to view v_ref
++drop cascades to materialized view m_ref
+ CONTEXT:  SQL statement "DROP TABLE alter_table_set_access_method.ref CASCADE"
+ NOTICE:  renaming the new table to alter_table_set_access_method.ref
+  alter_table_set_access_method
+ -------------------------------
+
+ (1 row)
+```
+
+**Fix 1**
+```diff
++\set VERBOSITY terse
+```
+
+**Example of issue 2**
+```diff
+SET citus.log_remote_commands TO ON;
+ -- should propagate to all workers because no table is specified
+ ANALYZE;
+ NOTICE:  issuing BEGIN TRANSACTION ISOLATION LEVEL READ COMMITTED;SELECT assign_distributed_transaction_id(0, 3461, '2022-08-19 01:56:06.35816-07');
+ DETAIL:  on server postgres@localhost:57637 connectionId: 1
+ NOTICE:  issuing BEGIN TRANSACTION ISOLATION LEVEL READ COMMITTED;SELECT assign_distributed_transaction_id(0, 3461, '2022-08-19 01:56:06.35816-07');
+ DETAIL:  on server postgres@localhost:57638 connectionId: 2
+ NOTICE:  issuing SET citus.enable_ddl_propagation TO 'off'
+ DETAIL:  on server postgres@localhost:57637 connectionId: 1
+-NOTICE:  issuing SET citus.enable_ddl_propagation TO 'off'
+-DETAIL:  on server postgres@localhost:xxxxx connectionId: xxxxxxx
+ NOTICE:  issuing ANALYZE
+ DETAIL:  on server postgres@localhost:57637 connectionId: 1
++NOTICE:  issuing SET citus.enable_ddl_propagation TO 'off'
++DETAIL:  on server postgres@localhost:57638 connectionId: 2
+ NOTICE:  issuing ANALYZE
+ DETAIL:  on server postgres@localhost:57638 connectionId: 2
+```
+
+**Example of fix 2**
+```diff
+ SET citus.log_remote_commands TO ON;
++SET citus.grep_remote_commands = '%ANALYZE%';
+```
+
+
+### Isolation test completes in different order
+
+**Issue**: The two sessions in the isolation test get unblocked at the same
```

Comment:
pretty minor:

I find this wording a little confusing for `Issue` description; `get unblocked at the same time` 

I think the issue is more like: `there is no strict ordering between steps as there is not explicit locking between them`.   

And then the outcome of the issue is what you write. (so, happy to have (variations of) both sentences)